test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/Common/Server/Default/DefaultEdmModel.cs

```diff
@@ -37,6 +37,8 @@ public static IEdmModel GetEdmModel()
             builder.EntitySet<StoredPI>("StoredPIs");
             builder.EntitySet<Subscription>("SubscriptionTemplates");
             builder.Singleton<StoredPI>("DefaultStoredPI");
+            builder.EntitySet<Bank>("Banks");

```

Comment:
Why do we need these here and also in the _CommonEndToEndDataModel.cs_?

Reply:
fixed..